code/src/Album.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+
+
+export const Album = (props) => {
+  console.log(props)
+  return (
+    <section className="card" >
+      <div className="image-icon">
+        <div className="icons">
+          <img className="heart" src="icons/heart.svg" alt="Heart" />
+          <img className="play" src="icons/play.svg" alt="Play" />
+          <img className="dots" src="icons/dots.svg" alt="Dots" />
```

Comment:
Maybe `like` and `more` would be better names for the images? Or having a link around so it's easy to understand that the used can click on these images?